### PR TITLE
add address level schema

### PR DESCRIPTION
**Pull request by leozc**
Add a json schema for reader to understand what should be included on a label object

### CODE REVIEW

File: doc/taxonomy.md [hunk 1]
```diff
@@ -143,3 +143,137 @@ The protocol has a fixed list of values defined at the data model (as shown belo
...
+
+### Label Level Schema
+
+```json
+{
+  "address": "string",
+  "name": "string",
+  "source": {
+    "type": "string",
+    "enum": ["Ground Truth", "Heuristic", "Machine Learning", "Research", "External"]
```
Comment (beingzy) [inline doc/taxonomy.md:156]: 1) https://github.com/microscopexyz/chainmetareader/blob/main/doc/taxonomy.md (contains"research") vs. 2) https://docs.microscopeprotocol.xyz/onboarding/data/source (doest not contain "research"). this difference, I do not know where or when we introduced. Anyway, it should open an other discussion thread to make them converge. Regardlessly, it should not impact this PR

Comment (beingzy) [reply doc/taxonomy.md:156]: @tamamatammy @zhanwu within Coinbase, is there significant portion of labels from "research"? To simplify the array of source-type, is it okay to this source into "ground truth" (if the research can draw solid conclusion) or "heuristics" (it utilizes the investigation based on supporting material to draw a likely conclusion)?

**Review by beingzy — APPROVED** [review]
LGTM.

**Review by zhanwux — APPROVED** [review]

**Review by zhanwu — APPROVED** [review]

**Comment by zhanwu** [comment]
You need to sign the commit before pushing it

File: doc/taxonomy.md (hunk 1 above)
Comment (leozc) [reply doc/taxonomy.md:156]: @beingzy we can update the array if needed 

@tamamatammy what should be in the array above? We certainly want to constrain the taxonomy

**Comment by leozc** [comment]
@zhanwu you mean I need to sign?

**Pull request merged**